docs/guides/composition-revisions.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Composition Revisions
+toc: true
+weight: 260
+indent: true
+---
+
+This guide discusses the use of "Composition Revisions" to safely make and roll
+back changes to a Crossplane [`Composition`][composition-type]. It assumes
+familiarity with Crossplane, and particularly with
+[Composition][composition-term].
+
+> Composition Revisions are an __alpha feature__. They are not yet recommended
+> for production use, and are disabled by default.
+
+A `Composition` configures how Crossplane should reconcile a Composite Resource
+(XR). Put otherwise, when you create an XR the selected `Composition` determines
+what managed resources Crossplane will create in response. Let's say for example
+that you define a `PlatformDB` XR, which represents your organisation's common
+database configuration of an Azure MySQL Server and a few firewall rules. The
+`Composition` contains the 'base' configuration for the MySQL server and the
+firewall rules that is extended by the configuration fo the `PlatformDB`.
+
+There is a one-to-many relationship between a `Composition` and the XRs that use
+it. You might define a `Composition` named `big-platform-db` that is used by ten
+different `PlatformDB` XRs. Usually, in the interest of self-service, the
+`Composition` is managed by a different team from the actual `PlatformDB` XRs.
+For example the `Composition` may be written and maintained by a platform team
+member, while individual application teams create `PlatformDB` XRs that use said
+`Composition`.
```

Comment:
I feel like it probably isn't worth introducing an additional concept here, but just wanted to note that we typically talk about app teams authoring XRCs rather than XRs directly. That being said, "creating an XR" could be thought of here in a general sense as "via a Claim, or directly". Don't think we need to make any change, just thought I'd mention :)

Reply:
Yeah, I had a similar thought here. I figured it was better to keep it simple in this context. As you say, it's not incorrect - the creation is just indirect.